### ISSUE TITLE
Pixi Accessibility is now working with IOS VoiceOver

### ISSUE THREAD

**Issue by scottmcdonnell** [pixijs/pixi-accessibility#1]
If you enable VoiceOver in IOS 9 the pixi-accessibility demo does not allow the buttons to be tapped/selected. Swipe to tab through the tab order does not work either.

This seems to be because the divs are only added  when Tab button is pressed and therefore not available to the screen reader from page load.
